src/components/Navbar/Navbar.jsx

```diff
@@ -32,9 +36,12 @@ const Navbar = () => {
           </button>
           <Link to="/wishlist">
             <FavoriteBorderIcon className="icon" />
+            {wishList.length!=0?(<span className='badge-icon'>{wishList.length===0?null:wishList.length}</span>):null}
```

Comment:
you can also use and use double check like this "!=="
wishlist.length!==0 && <span> 
and 
within span same , wishlist.length > 0 && wishlist.length